pkg/containers/containers.go

```diff
@@ -169,7 +173,8 @@ func getContainerIdFromCgroup(cgroupPath string) (string, string) {
 			id = strings.TrimPrefix(id, "libpod-")
 		}
 
-		if matched, _ := regexp.MatchString(`^[A-Fa-f0-9]{64}$`, id); matched {
+		regex := containerIdFromCgroupRegex
+		if matched := regex.MatchString(id); matched {
```

Comment:
why not using `sync.Once` for `regexp.MustCompile` instead of defining a global var? I think it would be cleaner.